scan/src/pages/AccountIndexPage.re

```diff
@@ -160,7 +160,12 @@ let make = (~address, ~hashtag: Route.account_tab_t) =>
     let totalBalance = availableBalance +. balanceAtStakeAmount +. rewardAmount +. unbondingAmount;
     let send = () => {
       switch (accountOpt) {
-      | Some(_) => dispatchModal(OpenModal(SubmitTx(SubmitMsg.Send(Some(address)))))
+      | Some({address: sender}) =>
+        if (sender == address) {
+          Window.alert("Why are you sending BAND to yourself?");
```

Comment:
Change blocking to a warning instead. For example, "Are you sure you want to send tokens to yourself?" with "Continue" and "Cancel" options.